rules/max-asserts.js

```diff
@@ -56,12 +56,16 @@ const create = context => {
 };
 
 const schema = [{
-	type: 'integer'
+	title: 'maximum number of assertions for each test',
+	type: 'integer',
+	minimum: 0,
+	default: 5
```

Comment:
When you define the defaults here, you also need to remove the default handling from the rule itself. For example: https://github.com/avajs/eslint-plugin-ava/blob/d812210ea6d21dba0aabfbbed1e7a60dbaec5c25/rules/max-asserts.js#L10